tests/integration/tier0/destructive/conversion-method/test_pre_registered.py

```diff
@@ -1,4 +1,8 @@
 def test_pre_registered_system_conversion(convert2rhel, pre_registered):
     with convert2rhel("-y --debug") as c2r:
-        c2r.expect("Conversion successful!")
+        c2r.expect("Continue with the system conversion?")
+        c2r.sendline("y")
+
+        c2r.expect("Continue with the system conversion?")
+        c2r.sendline("y")
```

Comment:
ahhh ok, gotcha I'll make this change! I have done this for the rest of the tests so Ill fix that too! Thank you!

and so when sending the -y does it send it twice for me already, because that's what I thought what was needed for the integration tests in this PR?

Reply:
ahh ok I see now, thank you for clearing this up for me and helping me out with this! will get this stuff done for the integration tests

Reply:
no worries